docs/api_reference/src/quantities.tex

```diff
@@ -3801,6 +3835,44 @@
 \end{codeblock}
 \end{itemdescr}
 
+\rSec3[qty.val.cmp]{Value comparison}
+
+\indexlibraryglobal{is_eq_zero}
+\indexlibraryglobal{is_neq_zero}
+\indexlibraryglobal{is_lt_zero}
+\indexlibraryglobal{is_gt_zero}
+\indexlibraryglobal{is_lteq_zero}
+\indexlibraryglobal{is_gteq_zero}
+\begin{itemdecl}
+friend constexpr bool is_eq_zero(const quantity& q) requires @\seebelownc@;
+friend constexpr bool is_neq_zero(const quantity& q) requires @\seebelownc@;
+friend constexpr bool is_lt_zero(const quantity& q) requires @\seebelownc@;
+friend constexpr bool is_gt_zero(const quantity& q) requires @\seebelownc@;
+friend constexpr bool is_lteq_zero(const quantity& q) requires @\seebelownc@;
+friend constexpr bool is_gteq_zero(const quantity& q) requires @\seebelownc@;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+Let \tcode{is_\placeholdernc{F}_zero} be the function name.
+
+\pnum
+\returns
+\tcode{is_\placeholdernc{F}(q <=> zero())}\irefcpp{compare.syn}.
```

Comment:
See it working at <https://godbolt.org/z/e6dPW9KKc>.
Since these functions only work for `quantity`, I made them friends.
This is so that they're not added to `::std` in the proposal.

Reply:
I thought it might be useful for other abstractions as well. For example, `std::chrono::duration`.

Also, I am not sure if spaceship usage in the implementation is a better solution than mine, but I see how it simplifies wording.

Reply:
Moreover, this requires `Rep` to be `totally_ordered` even for equality checks only. Should we require `totally_ordered` in `Representation` (besides `WeeklyRegular`)?

Reply:
> I thought it might be useful for other abstractions as well. For example, `std::chrono::duration`.

I thought that the implementation is overreaching (as a non-member function in `::std`).
And similarly to adding to other type's interfaces outside the proposal.
I would prefer such a change to be presented as a poll to LEWG.

> Moreover, this requires `Rep` to be `totally_ordered` even for equality checks only.

Good point.
I'll replace `<=>` with `@` so `==` is used as appropriate.

> Should we require `totally_ordered` in `Representation` (besides `WeeklyRegular`)?

Definitely not.
Complex numbers would stop working.